prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -178,7 +180,8 @@ class FHIRConverter(
                 // is properly recorded in the report file table with the correct sender
                 actionHistory.trackExternalInputReport(
                     report,
-                    BlobAccess.BlobInfo(format, blobUrl, blobDigest.toByteArray())
+                    BlobAccess.BlobInfo(format, blobUrl, blobDigest.toByteArray()),
+                    payloadName
```

Comment:
This change to carry over `payloadname` from the submissions API is needed to continue supporting the `sendOriginal` function. This adds the information to the root report being created here.